test/classes/deitz/class/scope_resolve_classes.good

```diff
@@ -1 +1 @@
-32
+12
```

Comment:
This reflects a change in which `y` is printed.  Was that deliberate?  I don't see it called out in the PR message.

```chapel
module OuterModule {
  var y = 3;

  class C {
    var x = 2;
  }

  module M {
    import OuterModule.C;
    var y = 1;
    proc C.foo() {
      writeln(y, x);
    }
...
```

Reply:
ping?  The PR message describes behavior changes only in the generation of error and warning messages, but this reflects a change in how the reference to `x` in `C.foo()` gets resolved.  Was this intentional?

Reply:
Uh-oh, sorry about the missed comment. I believe this is intentional; in particular, there has been discussion around secondary methods and scopes here: https://github.com/chapel-lang/chapel/issues/21560 (example 6 in particular).